docs/topics/api/ratings.rst

```diff
@@ -27,6 +27,7 @@ user has already posted a rating for the current version of an add-on.
 .. http:get:: /api/v4/ratings/rating/
 
     :query string addon: The :ref:`add-on <addon-detail>` id, slug, or guid to fetch ratings from. When passed, the ratings shown will always be the latest posted by each user on this particular add-on (which means there should only be one rating per user in the results), unless the ``version`` parameter is also passed.
+    :query string include_flags_for: The user id to show flags for. If the request is made with an authenticated user matching this parameter value, a ``flags`` property will be added to the response as described below in :ref:`ratings <rating-detail-object>`.
```

Comment:
Can't we combine it with `show_permissions_for` (possibly renaming it in the process) ? Or at least, use the same nomenclature ? I don't care that much about which verb to use, but we should make up our mind between `show` and `include` if we are to have 2 different parameters.

Reply:
Could change it to `show_flags_for`.  I considered combining it but it doesn't require the `addon` parameter to be present - it could be used for a list of all ratings for a user too.  If we _do_ want to replace `show_permissions_for` with a combined `show_flags_and_permissions_for`* then we'd either have to drop the requiring `addon` validation check or (unnecessarily) require it when you just wanted flag info. 

*(Worst. Name. Ever)

Reply:
That's a good point. Let's keep the 2 separate, `show_permissions_for` really needs an `addon`, it'd be awkward to let it work without. Since we've started with the word `show` I'd be in favor of following that, so `show_flags_for` gets my vote (I'm biased because I'm the one who introduced `show_permissions_for`, though :)

Reply:
https://github.com/mozilla/addons-server/pull/9988/commits/f4bf3d514e65fb61a6dafafc552539de0bcfadc3